rust/src/xenctrl.rs

```diff
@@ -117,9 +121,9 @@ impl Xenctrl {
       if ret != 1 || u32::from(info.domain) != dom_id {
         let error = self.get_last_error();
         if error.code == ErrorCode::None {
-          return Err(Error::new(ErrorCode::InvalidParam, ""));
+          return Err(Error::new(ErrorCode::InvalidParam, ""))
```

Comment:
Why this change?

Reply:
the `;`was useless, not necessary in this PR